ui/src/components/LoadingScreen/LoadingScreen.jsx

```diff
@@ -1,21 +1,16 @@
 import React from 'react';
 import logo from '../../img/mxcube_logo20.png';
 import loadingAnimation from '../../img/loading-animation.gif';
+import styles from './LoadingScreen.module.css';
 
-export default class LoadingScreen extends React.Component {
-  render() {
-    return (
-      <div id="loading">
-      <img className="logo" src={logo} alt="logo" />
-      <div>
-        <h3>Loading, please wait</h3>{' '}
-        <img
-          className="loader-init"
-          src={loadingAnimation}
-          alt="loading"
-        />
-      </div>
+function LoadingScreen() {
+  return (
+    <div className={styles.root}>
+      <img className={styles.logo} src={logo} width="159" alt="MXCuBE" />
```

Comment:
Moving the width from the CSS to the HTML to avoid layout shift in case the image file is bigger.